scripts/db_migrator.py

```diff
@@ -811,6 +825,10 @@ def version_2_0_2(self):
         This is the latest version for 202012 branch
         """
         log.log_info('Handling version_2_0_2')
+
+        if self.stateDB.keys(self.stateDB.STATE_DB, "FAST_REBOOT|system"):
+            self.migrate_config_db_flex_counter_delay_status()
```

Comment:
Why has this logic changed 202012 specific version? AFAIK, the change is needed for 202211 and above, so only branch specific versions should be changed. This will likely also require separate PRs for feature branches.

Also this is missing call to set a new version and also a placeholder function for new version.

Reply:
You are right, we need these changes for every branch starting from 202111, 202205, 202211, master
I will create a separate PR's

Reply:
Here are the PR's:

[[202211][db_migrator] Add migration of FLEX_COUNTER_DELAY_STATUS during 1911->2211 upgrade + fast-reboot. Add UT.](https://github.com/sonic-net/sonic-utilities/pull/2838)
[[202205][db_migrator] Add migration of FLEX_COUNTER_DELAY_STATUS during 1911->2205 upgrade + fast-reboot. Add UT.](https://github.com/sonic-net/sonic-utilities/pull/2870)
[[202111][db_migrator] Add migration of FLEX_COUNTER_DELAY_STATUS during 1911->2111 upgrade + fast-reboot. Add UT.](https://github.com/sonic-net/sonic-utilities/pull/2869)